src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -316,7 +316,7 @@ paths:
             type: string
```

Comment:
It's best that there is no default, but samples would be nice. But I think that combination is not possible?

Reply:
I don't see the posibility to add examples without OpenAPI adding it to the result. I have written some examples in the textual description.